docs/user-guide/src/introduction.md

```diff
@@ -4,12 +4,48 @@
 
 # Metal³
 
-Metal³ project (pronounced: Metal Kubed) exists to provide components
-that allow you to do bare metal host management for Kubernetes. Metal³
-works as a Kubernetes application, meaning it runs on Kubernetes and is
-managed through Kubernetes interfaces.
+The Metal³ project (pronounced: "Metal Kubed") provides components for bare
+metal host management with Kubernetes. You can enrol your bare metal machines,
+provision operating system images, and deploy Kubernetes clusters to them. From
+there, operating and upgrading your Kubernetes clusters can be handled by
+Metal³. Moreover, Metal³ is itself a Kubernetes application, so it runs on
+Kubernetes, and uses Kubernetes resources and APIs as its interface.
 
 Metal³ is one of the providers for the Kubernetes sub-project [Cluster
-API](https://github.com/kubernetes-sigs/cluster-api) and is focused on
-providing Kubernetes native APIs to manage Kubernetes clusters in a bare
-metal environment.
\ No newline at end of file
+API](https://github.com/kubernetes-sigs/cluster-api). Cluster API provides
+infrastructure agnostic Kubernetes lifecycle management, and Metal³ brings the
+bare metal implementation. This is paired with one of the components from the
+OpenStack ecosystem, [Ironic](https://wiki.openstack.org/wiki/Ironic), which
+uses PXE and IPMI to boot and install machines.
```

Comment:
PXE/IPMI, while popular, are slowly losing in interest in favour of Redfish/virtual media. To describe Ironic, I'd rather use something more generic, like "... which supports a wide range of bare metal management technologies and protocols". Or something like that. Use https://ironicbaremetal.org/ for inspiration.

Reply:
Good point. I've updated this to:

>... which supports a wide range of bare metal management protocols to boot and install machines, no matter the brand of hardware.

The "no matter the brand of hardware" bit comes from the website. What do you think?